bin/configs/php-ze-ph.yaml

```diff
@@ -1,4 +1,4 @@
 generatorName: php-ze-ph
 outputDir: samples/server/petstore/php-ze-ph
-inputSpec: modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
```

Comment:
I assume fake petstore cover edge cases not yet handled by this generator, which is ok.

In the medium, we plan to switch from petstore to something else.

Reply:
Yeah, I wanted to ask which 3.0 spec we should use from now, because I noticed links to different 3.0 files across generators.